src/core/audio/AudioSdl.cpp

```diff
@@ -107,11 +120,19 @@ AudioSdl::AudioSdl( bool & _success_ful, AudioEngine*  _audioEngine ) :
 	m_inputAudioHandle = m_audioHandle;
 	m_inputAudioHandle.callback = sdlInputAudioCallback;
 
-	m_inputDevice = SDL_OpenAudioDevice (nullptr,
-										 1,
-										 &m_inputAudioHandle,
-										 &actual,
-										 0);
+	const auto inputDevice = ConfigManager::inst()->value(SectionSDL, InputDeviceSDL);
+	const bool isDefaultInput = inputDevice.isEmpty();
+
+	// Try with the configured device
+	const auto inputDeviceCStr = isDefaultInput ? nullptr : inputDevice.toLocal8Bit().data();
```

Comment:
```suggestion
	const auto inputDevice = ConfigManager::inst()->value(SectionSDL, InputDeviceSDL).toStdString();
	const bool isDefaultInput = inputDevice.empty();

	// Try with the configured device
	const auto inputDeviceCStr = isDefaultInput ? nullptr : inputDevice.c_str();
```
Again, fix encoding and dangling pointer.

Reply:
Done with commit c1e5de5b3b9.